contracts/APPCoin.sol

```diff
@@ -24,6 +24,12 @@ contract APPCoin is Initializable, ERC777Upgradeable, PausableUpgradeable, Ownab
         require(msg.sender == apiCoin, 'ApiCoin Required');
         _mint(from, amount,'','');
     }
+
+    function setResourceWeightBatch(uint32[] calldata indexArr, string[] calldata resourceIdArr, uint[] calldata weightArr) onlyOwner public {
+        for(uint256 i=0; i<indexArr.length; i++) {
```

Comment:
Do we need to make sure `len(indexArr) = len(resourceIdArr) = len(weightArr)` here?

Reply:
Checked.